src/app/components/home-page/home-page.component.scss

```diff
@@ -0,0 +1,52 @@
+.content {
+  width: 100%;
+}
+
+.img-container {
+
+  display: flex;
+  flex-flow: column;
+  flex: 1 1 auto;
+  align-items: center;
+  justify-content: center;
+}
+img {
+  max-height: 35%;
+  max-width: 35%;
+  animation: pulse 3s infinite;
+  animation-direction: alternate;
+}
+
+@keyframes pulse {
```

Comment:
widze takie coś pierwszy raz ale pewnie dlatego że nie lubie animacji na stronkach :D

Reply:
okej to usuwać czy zostawić :D ?